src/lib/undux/GDStore.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import { createConnectedStore, withReduxDevtools, type StoreDefinition } from 'undux'
 import compose from 'lodash/fp/compose'
-
+import { type ZoomCaptureResult } from './Zoom'
```

Comment:
why is it required here?

Reply:
left over, removed. used to be part of the store and than we omitted store usage for communication between inner FR components

Reply:
removed again, commit 6ba7993227d9b85887297e882462928196ea00b9